src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,53 +4,54 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated to use the `COLORS` constant from the `utils/theme` module instead of the previous `colors` constant. This change is consistent and improves maintainability by centralizing color definitions in a single theme file.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->